suites/message/nested.go

```diff
@@ -210,23 +212,126 @@ func TestNestedSends(t *testing.T, factory state.Factories) {
 		td.AssertBalance(stage.creator, big.Sub(balanceBefore, result.Receipt.GasUsed.Big())) // Pay gas, don't receive funds.
 	})
 
+	// The multisig actor checks before attempting to transfer more than its balance, so we can't exercise that
+	// the VM also checks this. Need a custome actor to exercise this.
```

Comment:
Clarifying the thing we want to exercise is that inner sends for an amount greater than actor balance fail as expected, correct? In a similar vein to this, do we exercise inner sends of adequate balance for transfer but inadequate to cover gas cost?

Reply:
the inner sender doesn't pay for gas.